src/cli/commands/distinct.rs

```diff
@@ -12,10 +12,10 @@ use pastel::random::{self, RandomizationStrategy};
 
 pub struct DistinctCommand;
 
-fn print_iteration(out: &mut dyn Write, brush: &Brush, stats: &IterationStatistics) -> Result<()> {
+fn print_iteration(out: &mut Output, brush: &Brush, stats: &IterationStatistics) -> Result<()> {
```

Comment:
I'm sorry, but we don't need to change `dyn Write` to `Output` here. There's no need to wrap the `stderr` handle in Output as well. We are not calling `show_color` here.